### PR TITLE
try out this java code

### PR DESCRIPTION
**Pull request by gaurav-lomte23**
i have worked on this and this one's just bit easy to understand please merge this file

### CODE REVIEW

**Comment by gaurav-lomte23** [comment]
please check out the program once

**Comment by Pradyuman7** [comment]
Thanks for contributing, please take a look at contributing.md file to know more how to make a valid contribution. Please do not modify existing code and try to contribute something that's not already present

**Comment by gaurav-lomte23** [comment]
> Thanks for contributing, please take a look at contributing.md file to know more how to make a valid contribution. Please do not modify existing code and try to contribute something that's not already present

Please merge it I have added new file without changing any inital code

File: Matrix/Rotate90.java [hunk 1]
```diff
@@ -8,49 +8,23 @@ static void Reverse(int i, int mat[][], int N)
 		int start = 0;
 		int end = N - 1;
 
-		while (start < end) {
```
Comment (Pradyuman7) [inline Matrix/Rotate90.java:11]: This is the file which is being changed

**Review by Pradyuman7 — CHANGES_REQUESTED** [review]

**Comment by Pradyuman7** [comment]
Closing this PR for now, feel free to reopen if you want to work on it

**Pull request closed without merging**